tests/bootstrap.php

```diff
@@ -45,3 +46,8 @@
 }
 
 Type::addType('uuid', UuidType::class);
+
+// Ignore unfixable deprecations
+Deprecation::ignoreDeprecations(
+    'https://github.com/doctrine-extensions/DoctrineExtensions/pull/2772', // Ignore annotations deprecations from self
+);
```

Comment:
This is useful if you're running with `<env name="DOCTRINE_DEPRECATIONS" value="trigger"/>` in a custom `phpunit.xml` to see more of the triggered deprecations